spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :feature do
+  before do
+    @user1 = User.create!(id: 5, name: 'Khaing',
+                          photo: 'https://c0.klipartz.com/pngpicture/695/991/gratis-png-profesor.png', bio: 'officer')
+    @user2 = User.create!(id: 6, name: 'May',
+                          photo: 'https://c0.klipartz.com/pngpicture/695/991/gratis-png-profesor.png', bio: 'student')
+    Post.create!(title: 'Intro', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user1)
+    Post.create!(title: 'Intro1', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user1)
+    Post.create!(title: 'Intro', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user1)
+    Post.create!(title: 'Intro', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user1)
+    Post.create!(title: 'Intro', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user1)
+
+    @post1 = Post.create!(title: 'Intro2', text: 'Hi', comment_counter: 0, like_counter: 2, author: @user2)
+    @comment1 = Comment.create(post: @post1, user: @user2, text: 'My comment')
+    @like1 = Like.create(post: @post1, user: @user2)
+  end
+
+  describe 'returns user index page' do
+    it "returns user's profile" do
+      visit user_posts_path(@user2)
+      expect(page).to have_css("img[src*='#{@user2.photo}']")
+    end
+
+    it 'returns user name' do
+      visit user_posts_path(@user2)
+      expect(page).to have_content('May')
+    end
+
+    it 'return number of posts' do
+      visit user_posts_path(@user2)
+
+      expect(@user2.posts.count).to have_content(1)
+    end
+
+    it 'return pagination button' do
+      visit user_path(@user1)
+      click_link 'See all posts'
+      sleep 10
+      expect(page).to have_css('.paginate')
+    end
+
+    it "return post's title" do
+      visit user_posts_path(@user2)
+      expect(@user2.posts[0].title).to have_content('Intro')
+    end
+
+    it "return post's body" do
+      visit user_posts_path(@user2)
+      expect(@user2.posts[0].text).to have_content('Hi')
+    end
+
+    it 'return first comment' do
+      visit user_post_path(@user2, @post1)
+      expect(@user2.posts[0].recent_comments.map(&:text)).to include('My comment')
+    end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#work-and-submission-mode:~:text=the%20post%27s%20body.-,I%20can%20see%20the%20first%20comments%20on%20a%20post.,-I%20can%20see), This test should assert that you can see the first comments (more than one comment) on a post, not only the first comment as you did, So Kindly create more than one comment and assert there existence.